src/Components/Navbar/Navbar.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import "./Navbar.css";
+import { Link } from "react-router-dom"
+import NavLogo from "../../Assets/nav-logo.png"
+import { useFilter } from '../../Contexts/filter-controller'
+
+const Navbar = () => {
+    const { dispatch, filteredState } = useFilter()
```

Comment:
Suggestion: These are usused here so you can remove them

Reply:
done